terraform/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-This terraform folder creates all the required AWS services for this project bar the ECR's containing the images. This includes the AWS postgres RDS ...
+This terraform folder creates all the required AWS services for this project bar the ECR's containing the images. This includes the AWS postgres RDS, the extract lambdas , ....
 
 ## Set-up and Running
 
```

Comment:
Filling this in with the basic info about setup from the start is going to make your life easier later.